sig-wg-lifecycle.md

```diff
@@ -0,0 +1,116 @@
+# SUMMARY:
+
+This document covers everything you need to know about the creation and retirement (“lifecycle”) of a special interest or working group within the Kubernetes project. General project governance information can be found in the [steering committee repo].   
+Out of scope for this document: [subproject] creation.
+
+[Creation]  
+[Retirement]
+
+# [Creation]
+## Prerequisites for a SIG
+- [ ] Read [sig-governance.md]
+- [ ] Send an email to the Steering Committee <steering@kubernetes.io> to scope the SIG and get provisional approval.
+- [ ] Look at the checklist below for processes and tips that you will need to do while this is going on. It's best to collect this information upfront so you have a smoother process to launch
+- [ ] Follow the [SIG charter process] to propose and obtain approval for a charter
+- [ ] Announce new SIG on kubernetes-dev@googlegroups.com
+
+## Prerequisites for a WG
+- [ ] Read [wg-governance.md]
+- [ ] Send email to [kubernetes-dev@googlegroups.com] with some of the questions answered from wg-goverance.md and wait for community discourse; ask for SIG sponsorship
```

Comment:
Should we have a template for this so they are easy to spot.  e.g. the Subject is `WG-Creation-Request: WG Foo`